### ISSUE TITLE
running test-suite as root creates an empty /TEST directory

### ISSUE THREAD

**Issue by mbiebl** [systemd/systemd#5856]
### Submission type

  - [X] Bug report
  - [ ] Request for enhancement (RFE)

### systemd version the issue has been seen with

git master

### Used distribution

Debian sid

Running the test suite as root, creates an empty top-level /TEST directory

Steps to reproduce
```
# id
uid=0(root) gid=0(root) groups=0(root)
# meson build
# ninja -C build
# ninja -C build test
# ls -al /TEST/
total 8
drwxr-xr-x  2 root root 4096 Apr 29 13:17 .
drwxr-xr-x 23 root root 4096 Apr 29 13:17 ..
```

**Comment by evverx** [comment]
Some subtests of `test-execute` create `/TEST` and mount `tmpfs` onto it to check out the mount propagation. I think that is not related to `meson`.

**Comment by poettering** [comment]
uh, this is bad

**Comment by evverx** [comment]
The fix is waiting in #5943. Please take a look.
